MdeModulePkg/Bus/Pci/NvmExpressDxe/NvmExpress.h

```diff
@@ -4,6 +4,7 @@
 
   (C) Copyright 2016 Hewlett Packard Enterprise Development LP<BR>
   Copyright (c) 2013 - 2019, Intel Corporation. All rights reserved.<BR>
+  Copyright (c) Microsoft Corporation.

```

Comment:
Could you add the `<BR>` at the end for doxygen, please?

Reply:
Updated in the latest push. Thanks for reviewing.